### PR TITLE
Fix missing types

### PR DESCRIPTION
**Pull request by BESHOI**
add types to size-adjust, ascent-override;

### CODE REVIEW

File: packages/react/types/css.d.ts [hunk 1]
```diff
@@ -9303,6 +9303,8 @@ export namespace AtRule {
     fontWeight?: FontWeight;
     src?: string;
     unicodeRange?: string;
+		sizeAdjust?: string;
+  	ascentOverride?: string;
```
Comment (LucasUnplugged) [inline packages/react/types/css.d.ts:9307]: This was probably accidental, but you should try to maintain the existing indentation.

Comment (BESHOI) [reply packages/react/types/css.d.ts:9307]: oh, thanks i will

**Review by hadihallak — APPROVED** [review]
Thanks for the PR @BESHOI ❤️

**Comment by BESHOI** [comment]
Thank you so much @hadihallak 😍

**Pull request merged**